go.mod

```diff
@@ -1,17 +1,20 @@
 module github.com/flashbots/rpc-endpoint
 
-go 1.20
+go 1.21
```

Comment:
is it necessary to make the _minimum_ version v1.21?

Reply:
Ya, versions older than 1.21 choke on the new `toolchain` directive below that go 1.22 adds.  I'm actually surprised that the Go team didn't consider this, I feel like the toolchain directive should've only been added for newer `go` versions but I can't find a way to do that.

Basically running into the same issue as: https://github.com/golang/go/issues/65847#issuecomment-1957608903